lib/har/bestpractice/lastModifiedTime.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+module.exports = {
+  id: 'lastModifiedTime',
+  title: 'Avoid setting last modified header in the future',
+  description: 'Dont\'t set last modified headers in the future!',
+  weight: 0,
+  tags: ['performance', 'bestpractice', 'server'],
+
+  processPage: function(page) {
+    let score = 100;
+    let offending = [];
+    page.assets.forEach(function(asset) {
+      if (asset.timeSinceLastModified && parseInt(asset.timeSinceLastModified) < 0) {
```

Comment:
Thanks @XhmikosR for the feedback! I've neglected this pull request for a while. I'm going to try to pick it back up after the release of 4.0 of sitespeed.